docs/eventing/channels/create-default-channel.md

```diff
@@ -6,42 +6,61 @@ type: "docs"
 showlandingtoc: "true"
 ---
 
-# Creating a channel using cluster or namespace defaults
+Developers can create channels of any supported implementation type by creating an instance of a
+`Channel` object.
 
-Developers can create channels of any supported implementation type by creating an instance of a Channel object.
+To create a channel:
 
-1. Create a Channel object:
+1. Create a [Channel object](https://knative.dev/docs/reference/api/eventing/#messaging.knative.dev/v1.Channel): <!-- Where do you enter the below? -->
```

Comment:
Again, Channel vs `Channel` inconsistent, please check throughout and pick one to go with and add to the style guide. Pretty sure the k8s guide has guidance on formatting API objects we can use?

Reply:
re "Where do you enter the below?" you create a YAML file.
I'd update this to use the `kubectl apply -f <<EOF` rules we added to the [style guide](https://docs.google.com/document/d/1tigolRzKhronvgoAJfszE-XHWWd0LQ8o_j0D73O9KvM/edit#heading=h.qqz6in2pxqzk) - same for all bits where YAMLs are created please.

Reply:
Thanks, will change accordingly.